tests/compile_ouroboros.py

```diff
@@ -0,0 +1,670 @@
+# -*- coding: utf-8 -*-
```

Comment:
Actually, putting it in the "tools" subdirectory would be better. There's no analog in Batavia (yet), but there probably should be.